region/norfair/east.json

```diff
@@ -4531,10 +4573,118 @@
             ]
           },
           "locks": null
+        },
+        {
+          "id": 4,
+          "name": "Elevator to Lower Norfair",
+          "nodeType": "door",
+          "nodeSubType": "elevator",          
+          "locks": null
         }
       ],
       "enemies": [],
-      "links": []
+      "links": [
+        {
+          "from": 1,
+          "to": [
+            {
+              "id": 2,
```

Comment:
According to the picture, 1 connects only to 3

Reply:
I got to stop doing these things late at night.